frame.go

```diff
@@ -185,6 +186,18 @@ func (f *Frame) Service() string {
 	return string(f.Payload[_serviceNameIndex : _serviceNameIndex+l])
 }
 
+// isLast indicates whether the frame has the continuation bit set.
+func (f *Frame) isLast() bool {
+	switch f.messageType() {
+	case messageTypeCallReq, messageTypeCallRes, messageTypeCallResContinue, messageTypeCallReqContinue:
+		flags := f.Payload[_flagsIndex]
+		return (flags&hasMoreFragmentsFlag != hasMoreFragmentsFlag)
```

Comment:
nit: don't need parens here

also, normally for bitsets, i just check ` !- 0` or ` == 0`, little bit simpler